lib/util.js

```diff
@@ -589,7 +589,7 @@ define(["l10n","sugar-web/datastore","FileSaver"], function (l10n, datastore, Fi
 	// Load a file
 	util.loadFile = function(file, callback) {
 		// Use FileReaer object in web
-		if (util.getClientType() == constant.webAppType || constant.noServerMode || (util.getClientType() == constant.appType && !util.platform.android && !util.platform.ios && !util.platform.electron)) {
```

Comment:
You should not remove the `constant.noServerMode` test since it's not related to Android or iOS